src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoader.cs

```diff
@@ -180,7 +180,7 @@ bool GetForInnerPathExistence(KeyValuePair<string, JsonElement> o) =>
             MergeForkIdTransition = chainSpecJson.Params.MergeForkIdTransition,
             TerminalTotalDifficulty = chainSpecJson.Params.TerminalTotalDifficulty,
             TerminalPoWBlockNumber = chainSpecJson.Params.TerminalPoWBlockNumber,
-            BlobSchedule = new Dictionary<string, ChainSpecBlobCountJson>(chainSpecJson.Params.BlobSchedule, StringComparer.OrdinalIgnoreCase),
+            BlobSchedule = [.. chainSpecJson.Params.BlobSchedule],
```

Comment:
Do we want/need to convert to array? (maybe yes)

Reply:
It converts from array to a sorted set. Can be a sorted set in first place 🤔